### ISSUE TITLE
Support homebrew install

### ISSUE THREAD

**Issue by d1y** [tekumara/typos-lsp#66]

**Comment by janbridley** [comment]
Ideally, this package could be installed via [brew](https://brew.sh) on macOS, similar to [ruff-lsp](https://formulae.brew.sh/formula/ruff-lsp#default), [rust-analyzer](https://formulae.brew.sh/formula/rust-analyzer), and many others. This would make the package more accessible and drive new users to this implementation.

**Comment by tekumara** [comment]
Yes I agree! The brew maintainers [frown on authors submitting their own work](https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff), would someone be willing to give this a go?

**Comment by tekumara** [comment]
Yes I agree! The brew maintainers [frown on authors submitting their own work](https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff), would someone be willing to give this a go? Those existing rust projects look like a good starting point.

**Comment by janbridley** [comment]
I have added this to my TODOs, and will update when I have progress.

**Comment by tekumara** [comment]
Thank you @zyoshoka for [adding typos-lsp to Homebrew](https://github.com/Homebrew/homebrew-core/pull/197313)!
